frontend/packages/integration-tests-cypress/views/common.ts

```diff
@@ -1,2 +1,11 @@
+import { expect } from 'chai';
+
 export const resourceStatusShouldContain = (desiredStatus: string, options?: any) =>
   cy.contains('[data-test="status-text"]', desiredStatus, options);
+
+export const shouldMatchRegex = (selector: string, regex: any) => {
```

Comment:
Maybe should add `regex` to  `cy.byTestID(id: string, regex:any)` and put the regex handling in `cy.byTestID` if regex param detected.